src/PVRIptvData.cpp

```diff
@@ -428,22 +429,28 @@ bool PVRIptvData::LoadPlayList(void)
 
         if (!strGroupName.empty())
         {
-          strGroupName = XBMC->UnknownToUTF8(strGroupName.c_str());
+          std::stringstream stream(strGroupName);
```

Comment:
This overwrites `std::stringstream stream` from line 312. Gonna rename it to `streamGroup`